moveit_core/collision_detection/include/moveit/collision_detection/test_collision_common_pr2.h

```diff
@@ -309,7 +309,7 @@ TYPED_TEST_P(CollisionDetectorTest, AttachedBodyTester)
   shapes.push_back(shapes::ShapeConstPtr(shape));
   poses.push_back(Eigen::Isometry3d::Identity());
   std::vector<std::string> touch_links;
-  robot_state.attachBody("box", shapes, poses, touch_links, "r_gripper_palm_link");
+  robot_state.attachBody("box", poses[0], shapes, poses, touch_links, "r_gripper_palm_link");
```

Comment:
Note that I rely on the fact that all these tests are done with Identity transforms. Semantically it is incorrect to reuse this `poses[0]` for the second argument I think.

Reply:
Oh yeah, that will end up applying the first shape's pose twice, which doesn't make sense. In existing, the object pose is always the identity transform (since it is not set). I could keep the old API alive, but it feels like clutter in an interface that isn't facing the end user anyway. Maybe someone else has an opinion?

Reply:
Actually, since you can just pass in `Eigen::Isometry3d::Identity()` to fill the call, I would prefer to add a note in `MIGRATIONS.md` and just change the function, maybe using `[[deprecated]]`.

Reply:
I agree that keeping the old interface around only adds unnecessary clutter. I don't know the policy for interface changes.